week9/answers.sql

```diff
@@ -1,3 +1,129 @@
 -- Create your tables, views, functions and procedures here!
 CREATE SCHEMA social;
 USE social;
+
+-- Create Users Table
+CREATE TABLE users (
+  user_id INT PRIMARY KEY,
+  first_name VARCHAR(50),
+  last_name VARCHAR(50),
+  email VARCHAR(100),
+  created_on DATETIME DEFAULT CURRENT_TIMESTAMP
+);
+
+-- Create Sessions Table
+CREATE TABLE sessions (
+  session_id INT PRIMARY KEY,
+  user_id INT,
+  created_on DATETIME DEFAULT CURRENT_TIMESTAMP,
+  updated_on DATETIME DEFAULT CURRENT_TIMESTAMP,
```

Comment:
These should be TIMESTAMP or you will have undesired behavior with daylight savings time changes or any other changes that deviate from consistent UTC.

Reply:
"updated_on" is also missing the "ON UPDATE" clause.